### DISCUSSION TITLE
`remove_chimera` is ignored?

### DISCUSSION THREAD

**Discussion by gregdenay** [bio-raum/FooDMe2#78] [General]
Hi @marchoeppner 
i was lookingfor something else and stumbled upon this. It seems that the `remove_chimera BOOL` parameter is only present in the `nextflow.config` file and has no effect on workflow execution. Chimeras will always be filtered? 
Or did I miss some nextflow magic again?

**Comment by marchoeppner** [comment]
I like that you trust Nextflow so much that it borders on magic ;)

No, I think skipping the removal of chimera isn't currently implemented. We'd have to wrap the relevant steps (DADA2_RMCHIMERA and VSEARCH_UCHIME_DENOVO) into an if-else loop based on the state of params.remove_chimera  and connect the downstream processes when in the "skip" state. 

Right now, chimeras are always removed (which probably isn't the worst idea, but I suppose they may be reasons where one wants to skip it). 

/M

**Comment by gregdenay** [comment]
There are so many files in there I never know if I am looking at right place :)
I'll make this an issue for 1.1
